lib/project_types/theme/commands/common/root_helper.rb

```diff
@@ -35,6 +37,25 @@ def root_value(options, name)
           "."
         end
 
+        def theme_directory?(root)
+          Theme::Models::SpecificationHandlers::Theme.new(root).valid?
+        end
+
+        def current_directory_confirmed?
+          return false if @ctx.testing?
```

Comment:
why do we need this override when testing?

Reply:
> +1 for this approach

Great, CLI will `raise` an error now for this case :)